services/api-server/src/simcore_service_api_server/api/routes/files.py

```diff
@@ -59,12 +62,12 @@
 #
 #
 
-_COMMON_ERROR_RESPONSES: Final[dict] = {
+_FILE_STATUS_CODES: dict[int | str, dict[str, Any]] = {
     status.HTTP_404_NOT_FOUND: {
         "description": "File not found",
-        "model": ErrorGet,
-    },
-}
+        "model": HTTPExceptionModel,
```

Comment:
Why did you remove `ErrorGet`. Please note that this model is actually used and I see that it is different from `HTTPExceptionModel`

Reply:
I don't think `ErrorGet` can be raised in any of the methods in this file. The HTTPExceptions which can be raised within the methods in this file all have the `HTTPExceptionModel` (as far as I can tell, but maybe I am missing something?)

Reply:
OK, I missed the fact that there was a handler handling the `HTTPExceptions`. This would mean that all of my exceptions are caught by that handler as well. Hence, I reintroduce the `ErrorGet` class and use it in my exceptions as well.